ntoskrnl/ps/query.c

```diff
@@ -212,9 +212,13 @@ NtQueryInformationProcess(
             /* Get additional information, if needed */
             if (Extended)
             {
+                QuotaLimits.Flags |= (Process->Vm.Flags.MaximumWorkingSetHard ?
+                    QUOTA_LIMITS_HARDWS_MAX_ENABLE : QUOTA_LIMITS_HARDWS_MAX_DISABLE);
+                QuotaLimits.Flags |= (Process->Vm.Flags.MinimumWorkingSetHard ?
+                    QUOTA_LIMITS_HARDWS_MIN_ENABLE : QUOTA_LIMITS_HARDWS_MIN_DISABLE);
+
```

Comment:
Are there numerical members in the QuotaLimits structure associated with the presence of these flags?

Reply:
I think these flags control the enforcement of `QUOTA_LIMITS::MinimumWorkingSetSize` and `QUOTA_LIMITS::MaximumWorkingSetSize`, according to this.
[https://learn.microsoft.com/en-us/windows/win32/api/memoryapi/nf-memoryapi-getprocessworkingsetsizeex](https://learn.microsoft.com/en-us/windows/win32/api/memoryapi/nf-memoryapi-getprocessworkingsetsizeex)